modules/eventing/pages/eventing-examples-high-risk.adoc

```diff
@@ -2,129 +2,671 @@
+  "date": "2018-09-14T20:46:10+05:30",
+  "currency": "GBP",
+  "comments": "High Risk Txn: currency mismatch card: USD != txn: GBP",
+  "reason_code": "X-MISMATCH"
+}
+----
 +
-This indicates that transactions which were flagged as high risk gets moved to the _flagged_transactions_ bucket.
+image::high_risk_txns_08_bucket_documents_edit.png[,578]
 
-. From the *Couchbase Web Console* > *Query* page, execute the below N1QL query:
+. From the *Couchbase Web Console* > *Query* page, run a few N1QL queries on the new documents create in the *'review*' bucket:
++
+*QUERY A*, see how many high-risk transactions we found
++
+[source,N1QL]
+----
+SELECT COUNT(*) num_high_risk FROM review WHERE type='transaction';
+----
++
+image::high_risk_txns_09_n1ql_a.png[,%100]
++
+*QUERY B*, look at all the data and with a specific order
++
+[source,N1QL]
+----
+SELECT * FROM review WHERE type='transaction' ORDER BY currency, amount DESC;
+----
++
+image::high_risk_txns_09_n1ql_b.png[,%100]
++
+*QUERY C*, look at the summarized data and with a specific order and grouping
++
+[source,N1QL]
+----
+SELECT COUNT(*) count, reason_code, SUM(amount) total_amount, currency 
+FROM review WHERE type='transaction' 
+GROUP BY reason_code, currency;
+----
++
+image::high_risk_txns_09_n1ql_c.png[,%100]
++
+*QUERY D*, look at a transaction record or document by key.
++
+[source,N1QL]
+----
+SELECT * FROM register USE KEYS ('transaction:tx-1505315650-403');
+----
++
+image::high_risk_txns_09_n1ql_d.png[,%100]
++
+*QUERY E*, look at a card record or document by key.
++
+[source,N1QL]
+----
+SELECT * FROM register USE KEYS ('card:4273-6623-8686-4599');
+----
++
+image::high_risk_txns_09_n1ql_e.png[,%100]
++
+*QUERY F*, look at a flagged transaction record or document that we wrote by key.
 +
 [source,N1QL]
 ----
-SELECT reason_code, COUNT(1), num_txns, SUM(amount) amount
-FROM `flagged_transactions`
-GROUP BY reason_code;
+SELECT * FROM review USE KEYS ('transaction:tx-1505315650-403');
+----
++
+image::high_risk_txns_09_n1ql_f.png[,%100]
+
+. The next step is to follow the Function logic in detail, to do this 'flush' all the generated documents in review, access 
+the *Couchbase Web Console* > *Buckets* page. You should see 40 documents in the '*review*' bucket.  Select the bucket '*review*' to expand the buckets view. Next click on the *'Flush'* button in a while the document count will drop to 0.
++
+image::high_risk_txns_10_flush_review.png[,%100]
+
+. From the *Couchbase Web Console* > *Eventing* page, click *Pause*.
++
+image::high_risk_txns_10_pause.png[,%100]
++
+** In the *Confirm Pause Function* dialog, click *Pause Function*.
++
+image::high_risk_txns_10_pause_confirm.png[,344]
++
+** The Eventing function is paused in a few seconds and can be edited. Click the *'Edit JavaScript'* button.
+** In the Editor dialog change the OnUpdate handler code from *'var verbose = 0'* to *'var verbose = 3'* you are only modifying line four of the *high_risk_txrns" Function as below:
```

Comment:
```suggestion
** In the Editor dialog change the OnUpdate handler code from `var verbose = 0` to `var verbose = 3` -- you are only modifying line four of the *high_risk_txns* function as below:
```
Again, use formatting rather than quotes, so the users do not enter the quotes.